src/Tizen.Account.AccountManager/Tizen.Account.AccountManager/AccountErrorFactory.cs

```diff
@@ -182,15 +183,14 @@ internal static Exception CreateException(AccountError err, string msg)
                     }
                 case AccountError.XMLFileNotFound:
                     {
-                        exp = new System.IO.FileNotFoundException(msg + " XML File not found");
+                        exp = new FileNotFoundException(msg + " XML File not found");
```

Comment:
```suggestion
                        exp = new FileNotFoundException($"{msg} XML File not found");
```

Reply:
This is fixed. Submitted for merge in pull request - https://github.com/Samsung/TizenFX/pull/756